src/components/Input.js

```diff
@@ -0,0 +1,162 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled/macro';
+
```

Comment:
This file can be renamed as `styles.js` as it contains all the styles. And you can do name export for every component style